src/iqm/iqm_client/iqm_client.py

```diff
@@ -627,6 +628,11 @@ def get_quantum_architecture(self, *, timeout_secs: float = REQUESTS_TIMEOUT) ->
             qa = QuantumArchitecture(**result.json()).quantum_architecture
         except (json.decoder.JSONDecodeError, KeyError) as e:
             raise ArchitectureRetrievalError(f'Invalid response: {result.text}, {e}') from e
+
+        # HACK add cc_prx, remove when DQA works
```

Comment:
```suggestion
        # TODO HACK add cc_prx, remove when DQA works
```
So that it can be found with a regex by others.

Reply:
Added.